tests.bzl

```diff
@@ -0,0 +1,552 @@
+# In order to generate the following list, on unix systems, you can
+# run  the following command:
+# echo "directories = [" && find . -maxdepth 1 -type d | sort | sed '1d; s/^\.\//    "/; s/$/",/' && echo "]"
+directories = [
```

Comment:
this is slightly hacky but will simplify maintenance:

`glob(["*"], exclude_directories=0, exclude=glob(["*"], exclude_directories=1))`